### PR TITLE
middle-section setup

### PR DESCRIPTION
**Pull request by maroun7**
Update index.html
--Create work section
--Create cards section
--Fill card with project info
--Create about myself section
--Add about myself info
--Fix Webhint error

Update style.css
--Add styles to sections
--Fix Stylelint error

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,23 +35,178 @@ <h1 class="primary-text">I’m Maroun Glad to see you!</h1>
...
+        Hello I’m a software developer!
+        I can help you build a product ,
+        feature or website Look through
+        some of my work and experience!
+        If you like what you see and have a
+        project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <h3 class="met">LET'S CONNECT</h3>
+
```
Comment (Shaher-11) [inline index.html:168]: kindly add some space between the paragraph and the h3 tag 
wrong2

File: index.html [hunk 2]
```diff
@@ -35,23 +35,178 @@ <h1 class="primary-text">I’m Maroun Glad to see you!</h1>
...
+          <ul class="tags2">
+            <li class="lang-block"><p class="lang-text">html</p></li>
+            <li class="lang-block"><p class="lang-text">css</p></li>
+            <li class="lang-block"><p class="lang-text">javaScript</p></li>
+          </ul>
+
+          <button type="button" class="card-button">See Project</button>
+        </div>
+      </div>
```
Comment (Shaher-11) [inline index.html:152]: fix the overlapping of the text as shown in the screenshot below

wrong1

File: index.html [hunk 3]
```diff
@@ -35,23 +35,178 @@ <h1 class="primary-text">I’m Maroun Glad to see you!</h1>
...
```
Comment (Shaher-11) [inline index.html:125]: fix the overlapping of the text as shown in the screenshot below

wrong1

File: index.html [hunk 4]
```diff
@@ -35,23 +35,178 @@ <h1 class="primary-text">I’m Maroun Glad to see you!</h1>
...
```
Comment (Shaher-11) [inline index.html:99]: fix the overlapping of the text as shown in the screenshot below

wrong1

File: index.html [hunk 5]
```diff
@@ -35,23 +35,178 @@ <h1 class="primary-text">I’m Maroun Glad to see you!</h1>
...
```
Comment (Shaher-11) [inline index.html:72]: fix the overlapping of the text as shown in the screenshot below

wrong1

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @maroun7 

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Kindly check the comments below.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @maroun7,

##### Great work on the project so far:clap::clap::clap:
however, the changes requested from the last review aren't done yet, 
so we need to implement those changes to get the project approved:muscle::muscle::muscle:

 ## Required Changes ♻️

- [ ] PLease checks the comments from the last review and fix the styles so it look like the original design.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: index.html (hunk 1 above)
Comment (maroun7) [reply index.html:168]: image
what's wrong in here

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello  @maroun7 🔥 

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

nice

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**